KarhooUISDK/CommonUI/UIDimens.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  UIDimens.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 05.01.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+// Note: When updating the UI, aproximate the commented values to the ones before each comment. E.g.: replace any spacing of value 17 with .standard (which is 16)
+public struct UIDimens {
+    public struct Spacing {
+        public static let xxxSmall: CGFloat = 1
+        public static let xxSmall: CGFloat = 2
+        public static let xSmall: CGFloat = 4 //3...5
+        public static let small: CGFloat = 8 //6...10
+        public static let medium: CGFloat = 12 //11...14
+        public static let standard: CGFloat = 16 //15...21
+        public static let large: CGFloat = 24 //22...28
+        public static let xLarge: CGFloat = 32 //29...40
+        public static let xxLarge: CGFloat = 44 //41...56
+        public static let xxxLarge: CGFloat = 64 // 57+
+        public static let marginToEdgeOfScreen: CGFloat = 20
+    }
+    
+    public struct Size {
```

Comment:
I guess more accurate naming'd be `Height`

Reply:
It's not always used for height. Sometimes the same value can be used for width as well. If we change it we'd anchor a button with height = UIConstants.Height.Button.standard and width = UIConstants.Height.Button.standard

Reply:
I modified the name of the struct from `Size` to `Dimension`